en/1-who-we-are.md

```diff
@@ -3,13 +3,14 @@
 
 Meeds is a collaborative effort led by diverse open-source software (OSS) communities dedicated to creating a decentralized digital world that resists centralized corporate control and promotes a free and open environment.
 
-The Meeds project uses the Ethereum ecosystem to implement the WoM protocols and manage the supply of the MEED token. This approach has helped prioritize security, accessibility, usability, and cost-effectiveness, making it attractive to a broad user base. However, the WoM is designed to be an autonomous and self-sufficient public good, aiming to become multichain. 
+We use the Ethereum ecosystem to implement the WoM protocols and manage the supply of the MEED token. This approach has helped us prioritize security, accessibility, usability, and cost-effectiveness, making it attractive to a broad user base. However, the WoM was designed to be an autonomous and self-sufficient public good, aiming to become multichain. 
 
 The legal entity behind Meeds is the _Meeds Association_, a non-profit organization registered under Swiss law in Geneva. It is responsible for regulating the issuance of MEED tokens and contracting with service providers. 
 
 The Meeds Association intends to progressively hand over control of the Meeds protocol and treasuries to the _Meeds DAO_, which is the decentralized autonomous organization that represents the decentralized governance arm of the Meeds ecosystem, empowering the community to participate in decision-making, proposal voting, and protocol upgrades.
 
-The relationship between the Meeds Association and Meeds DAO involves how they operate in the real world. Meeds Association is a traditional organization that have members, a board of directors, and operates under specific legal and regulatory frameworks. In the real world, Meeds Association may engage in activities such as organizing events, providing services, and advocating for its members. It may also have a physical office and staff members. Meeds DAO, on the other hand, operates without a central authority and make decisions through consensus of its members using blockchain-based voting mechanisms. It may also engage in activities such as funding projects, managing assets, and making decisions through smart contracts.
+The relationship between the Meeds Association and the Meeds DAO involves how they operate in the real world. The Association is a traditional organization with members and a board of directors under specific legal and regulatory frameworks. It may organize events, provide services, and advocate for its members. It may also have a physical office and staff members. 
+Meeds DAO, on the other hand, operates without a central authority and makes decisions through members' consensus using blockchain-based voting mechanisms. It may also engage in activities such as funding projects, managing assets, and making decisions through smart contracts.
```

Comment:
```suggestion
Meeds DAO, on the other hand, operates without a central authority and makes decisions through the consensus of its members using blockchain-based voting mechanisms. It may also engage in activities such as funding projects, managing assets, and applying decisions through smart contracts.
```